build.js

```diff
@@ -90,6 +91,7 @@ const context = await esbuild.context({
     minify: production,
     nodePaths,
     outdir,
+    ...(args.metafile && { metafile: true }),
```

Comment:
This is overly complicated.  Just let it be set to false...

Reply:
`metafile: !!args.metafile`?

Reply:
Done